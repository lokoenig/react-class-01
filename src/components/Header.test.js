import React from "react";
import ShallowRenderer from 'react-test-renderer/shallow';

import PageHeader from "./Header";

test('Confirm DOM structure of page header', ()=>{
    const renderer = new ShallowRenderer();
    renderer.render(<PageHeader />);

    expect(renderer.getRenderOutput()).toMatchSnapshot();
})
