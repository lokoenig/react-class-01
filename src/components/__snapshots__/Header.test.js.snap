// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Confirm DOM structure of page header 1`] = `
<header
  className="App-header"
>
  <img
    alt="logo"
    className="App-logo"
    src="logo.svg"
  />
</header>
`;
